task1/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+class Program
+{
+    static void Main()
+    {
+        if (!test.testing.test())
+        {
+            Console.WriteLine("Error!");
+        }
+    }
+}
```

Comment:
```suggestion
if (!test.testing.test())
{
    Console.WriteLine("Error!");
}
```
И, наверное, надо бы что-то писать, если всё ок.